releasenotes/notes/fix-thread-safe-cache-b3ccf646feb364c6.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixed an issue that could have caused an exception as a result of a
```

Comment:
Can we call out the impact to library and different products. What symptoms does this fix address

Reply:
that's hard to say really, as the `@cached` decorator is used in many places, so potentially it affected all products